keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/LoadBalancer.kt

```diff
@@ -0,0 +1,20 @@
+package com.netflix.spinnaker.keel.clouddriver.model
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter
+import com.fasterxml.jackson.annotation.JsonAnySetter
+import com.fasterxml.jackson.annotation.JsonIgnore
+
+class LoadBalancer {
+  @JsonIgnore
+  private val properties = mutableMapOf<String, Any>()
```

Comment:
If you look at the "model" in `clouddriver`, it's empty.

My approach here is to `objectMapper.convert()` these to the appropriate class from the aws sdk.